frontend/src/pages/org/collection-detail.ts

```diff
@@ -190,6 +293,35 @@ export class CollectionDetail extends LiteElement {
             ${msg("Edit Collection")}
           </sl-menu-item>
           <sl-divider></sl-divider>
+          ${!this.collection?.isPublic
+            ? html`
+                <sl-menu-item
+                  style="--sl-color-neutral-700: var(--success)"
+                  @click=${() => this.onTogglePublic(true)}
+                >
+                  <sl-icon name="globe2" slot="prefix"></sl-icon>
+                  ${msg("Make Public")}
+                </sl-menu-item>
+              `
+            : html`
+                <sl-menu-item style="--sl-color-neutral-700: var(--success)">
+                  <sl-icon name="box-arrow-up-left" slot="prefix"></sl-icon>
```

Comment:
Could we change to the right arrow variant of this and also swap it out in the other areas of the app that use menu items as links (technically not ideal) later?  I like the box arrows better for going elsewhere but left signifies going back.

<img width="249" alt="Screenshot 2023-08-02 at 5 18 33 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/a4dea5f6-1de2-40d0-9ae2-55209171e6ff">

```suggestion
                  <sl-icon name="box-arrow-up-right" slot="prefix"></sl-icon>
```

Reply:
Done in separate commit